node/README.md

```diff
@@ -1,24 +1,60 @@
-### plaid-node quickstart
+# Quickstart for plaid-node
 
-[Quickstart guide](https://plaid.com/docs/quickstart)
+To run this application locally, first install it and then run either of the flows described below. Additionally, please also refer to the [Quickstart guide](https://plaid.com/docs/quickstart).
 
+## Installing the quickstart app
 ``` bash
 git clone https://github.com/plaid/quickstart.git
 cd quickstart/node
 npm install
+```
 
+## The canonical flow
+``` bash
 # Start the Quickstart with your API keys from the Dashboard
 # https://dashboard.plaid.com/account/keys
 #
-# PLAID_PRODUCTS is a comma-separated list of products to use when initializing
-# Link. Note that this list must contain 'assets' in order for the app to be
-# able to create and retrieve asset reports.
+# PLAID_PRODUCTS is a comma-separated list of products to use when
+# initializing Link. Note that this list must contain 'assets' in
+# order for the app to be able to create and retrieve asset reports.
+
 PLAID_CLIENT_ID='CLIENT_ID' \
 PLAID_SECRET='SECRET' \
 PLAID_PUBLIC_KEY='PUBLIC_KEY' \
 PLAID_ENV='sandbox' \
 PLAID_PRODUCTS='transactions' \
 PLAID_COUNTRY_CODES='US,CA,GB,FR,ES' \
 node index.js
+
+# Go to http://localhost:8000
+```
+
+## The OAuth redirect flow
+Some European institutions require an OAuth redirect authentication flow, where the end user is redirected to the bank’s website or mobile app to authenticate. For this flow, you should provide two additional configuration parameters, PLAID_OAUTH_NONCE and PLAID_OAUTH_REDIRECT_URI.
+
+``` bash
+# Start the Quickstart with your API keys from the Dashboard
+# https://dashboard.plaid.com/account/keys
+#
+# PLAID_PRODUCTS is a comma-separated list of products to use when
+# initializing Link.
+#
+# You will need to whitelist the PLAID_OAUTH_REDIRECT_URI for
+# your client ID through the Plaid developer dashboard at
+# https://dashboard.plaid.com/team/api.
+#
+# Set PLAID_OAUTH_NONCE to a unique identifier such as a UUID.
+# The nonce must be at least 16 characters long.
```

Comment:
I think the README would flow more naturally if the `PLAID_OAUTH_REDIRECT_URI` and `PLAID_OAUTH_NONCE` comments were moved up to the `OAuth redirect flow` section above.

Reply:
https://github.com/plaid/quickstart/pull/91/commits/657f5715332db729de91d59b5b1df36e84dfa7ff